lms/djangoapps/certificates/models.py

```diff
@@ -176,3 +177,8 @@ def certificate_status_for_student(student, course_id):
     except GeneratedCertificate.DoesNotExist:
         pass
     return {'status': CertificateStatuses.unavailable, 'mode': GeneratedCertificate.MODES.honor}
+
+
+class CertificateGenerationConfiguration(ConfigurationModel):
+    """Configure certificate generation."""
+    pass
```

Comment:
We are defining specific model for the generate-cert configuration. Thats' why subclass it from ConfigurationModel.